whois-rpsl/src/main/java/net/ripe/db/whois/common/rpsl/attrs/Inet6numStatus.java

```diff
@@ -99,4 +99,8 @@ public boolean isValidOrgType(final OrgType orgType) {
     public String toString() {
         return literalStatus.toString();
     }
+
+    public boolean isAssignment(){
```

Comment:
If you are adding this function in Inet6num , it should also be added in Inetnum ?